functions/helpers/firebase.js

```diff
@@ -0,0 +1,25 @@
+const firebase = require("firebase");
+// Required for side-effects
+require("firebase/firestore");
+firebase.initializeApp({
```

Comment:
Use env variables instead of hard coded strings. (e.g. `process.env.ENVIRONMENT_VARIABLE`)

Reply:
Nice catch